src/app/layout/header/dialog-user/dialog-user.component.spec.ts

```diff
@@ -57,13 +56,14 @@ describe('DialogUserComponent', () => {
     const data = {
       name: 'name',
       address: '123',
-      birthdate: date.getTime()
+      birthdate: date.getTime(),
     };
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
I would prefer to disable eslint in spec files. I do not want to mock everything to fit the types or always put this eslint-disable.